### PR TITLE
[Ready2Merge] Fluidsynth

### PR DESCRIPTION
**Pull request by stefan-balke**

### CODE REVIEW

File: recipes/fluidsynth/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - pkg-config
+    - glib
+    - libsndfile
```
Comment (marcelotrevisani) [inline recipes/fluidsynth/meta.yaml]: Please remove ``glib`` and ``libsndfile`` from ``build`` requirements

File: recipes/fluidsynth/build.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir build
+cd build
+cmake .. -Denable-framework=OFF -DLIB_SUFFIX="" -Denable-libsndfile=ON -DCMAKE_INSTALL_PREFIX=$PREFIX
```
Comment (marcelotrevisani) [inline recipes/fluidsynth/build.sh]: I think you will need to also add the option ``-DCMAKE_PREFIX_PATH:PATH=$PREFIX``

**Comment by stefan-balke** [comment]
All comments resolved. PR is ready for now.

**Comment by stefan-balke** [comment]
Question: How could I add different versions? Is this just controlled by the build variable?

**Pull request merged**